src/openrct2-ui/windows/TileInspector.cpp

```diff
@@ -2369,3 +2377,10 @@ void WindowTileInspectorClearClipboard()
     if (window != nullptr)
         static_cast<TileInspector*>(window)->ClearClipboard();
 }
+
+void WindowTileInspectorKeyboardShortcutToggleInvisibility()
```

Comment:
You are duplicating code here - you’re using WindowFindByClass() both here and in `ShortcutToggleVisibility()`.

I suggest moving this code to `ShortcutToggleVisibility()` and getting rid of `WindowTileInspectorKeyboardShortcutToggleInvisibility()`. (Assuming a team member agrees.)

Reply:
You're definitely right that there's redundant code. But I am not sure how to remove the reliance on `WindowTileInspectorKeyboardShortcutToggleInvisibility`. I can, however, remove `ShortcutToggleVisibility` entirely and have this function instead work like the ride construction shortcuts.

Reply:
That works too :+1: